model/host/host.go

```diff
@@ -363,8 +363,8 @@ func (h *Host) IsEphemeral() bool {
 }
 
 func (h *Host) SetStatus(status, user string, logs string) error {
-	if h.Status == evergreen.HostTerminated {
-		msg := fmt.Sprintf("not changing the status of terminate host %s to %s", h.Id, status)
+	if h.Status == evergreen.HostTerminated && h.Provider != evergreen.ProviderNameStatic {
+		msg := fmt.Sprintf("not changing the status of terminated host %s to %s", h.Id, status)
```

Comment:
If it's not too much trouble, could you make these message.fields?

Reply:
Done.